x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -246,7 +246,7 @@ export function getWaterfall(
     };
   }
 
-  const waterfallItems = trace.map(traceItem => {
+  const waterfallItems = [...trace.items].map(traceItem => {
```

Comment:
Any reason for not doing this?
```suggestion
  const waterfallItems = trace.items.map(traceItem => {
```

Reply:
whoops, i forgot the change this back, when i removed the root fetch